mlflow/environment_variables.py

```diff
@@ -194,3 +194,7 @@ def get(self):
 MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT = _EnvironmentVariable(
     "MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT", int, None
 )
+
+MLFLOW_PYTORCH_DISABLE_AUTOMATIC_GPU_USAGE = _BooleanEnvironmentVariable(
```

Comment:
Before the PR is all wrapped up, can we add a quick message in the Pytorch docs about how to disable this feature by referring to this env variable? 
The env variable docs are pretty buried in the docs and most users are unaware of where to find them / what to look for.
Probably just a ..note: block here https://www.mlflow.org/docs/latest/models.html#pytorch-pytorch

Reply:
Great point - doing that